src/Uno.UI.Tests/Windows_UI_XAML_Controls/Frame/Given_Frame.cs

```diff
@@ -57,4 +173,26 @@ public void When_RemovedPage()
 	class MyPage : Page
 	{
 	}
+
+	class FirstPage : Page
+	{
+	}
+
+	class SecondPage : Page
+	{
+	}
+
+	class ThirdPage : Page
+	{
+	}
+
+	class DisallowNavigatingFromPage : Page
+	{
+		public static bool NavigatingFromCalled = false;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change the visibility of 'NavigatingFromCalled' or make it 'const' or 'readonly'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5305538)